src/mysky/index.ts

```diff
@@ -453,7 +453,7 @@ export class MySky {
    * @returns - The entry data.
    * @throws - Will throw if the length of the data is > 70 bytes.
    */
-  async setEntryData(path: string, data: Uint8Array, customOptions: CustomSetEntryOptions): Promise<EntryData> {
+  async setEntryData(path: string, data: Uint8Array, customOptions?: CustomSetJSONOptions): Promise<EntryData> {
```

Comment:
What does defaultSetJSONOptions get us here over defaultSetEntryOptions?

Reply:
`CustomSetJSONOptions` contains get entry options as well as set entry options, so the user can set both parts of the operation. It's not the best name for it, but let's not get caught up in a minor thing that may end up being temporary.